tests/unit_node/process_test.ts

```diff
@@ -390,6 +390,8 @@ Deno.test({
 Deno.test({
   name: "process.env",
   fn() {
+    assert(Object.prototype.hasOwnProperty(process, "env"));
```

Comment:
probably this need to be:
```suggestion
    assert(Object.prototype.hasOwnProperty.call(process, "env"));
```

BTW I think we can disable lint rule as these are test code